src/sentry/static/sentry/app/components/lastCommit.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import Avatar from './avatar';
+import TimeSince from './timeSince';
+
+import {t} from '../locale';
+
+const LastCommit = React.createClass({
+  propTypes: {
+    lastCommit: React.PropTypes.object.isRequired,
+    headerClass: React.PropTypes.string
+  },
+
+  renderMessage(message) {
+    if (!message) {
+      return t('No message provided');
+    }
+
+    if (message.length > 100) {
+      let truncated = message.substr(0, 90);
+      let words = truncated.split(' ');
+      // try to not have elipsis mid-word
+      if (words.length > 1) {
+        words.pop();
+        truncated = words.join(' ');
+      }
+      return truncated + '...';
```

Comment:
We should also consider the case where someone has a commit message like:

```
Fixed a thing ...

* Change #1
* Change #2
```

So, I'd probably split on `\n` first, then take first 90 chars. Otherwise we can get:

```
Fixed a thing ... * Change #1 * Change ...
```